android/src/main/kotlin/com/rokt/rokt_sdk/RoktWidgetFactory.kt

```diff
@@ -10,8 +10,8 @@ import io.flutter.plugin.platform.PlatformViewFactory
 class RoktWidgetFactory(private val messenger: BinaryMessenger): PlatformViewFactory(StandardMessageCodec.INSTANCE) {
```

Comment:
Just a question, the class RoktSdkPlugin which creates RoktWidgetFactory is ActivityAware. Could we pass context from RoktSdkPlugin to here to avoid `!!` and return that context if the context of the create function is null?

Reply:
No, onAttachedToEngine is where we create the RoktWidgetFactory which is called before onAttachedToActivity.